runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/DataflowOperationContext.java

```diff
@@ -296,55 +267,6 @@ public void reportLull(Thread trackedThread, long millis) {
       DataflowWorkerLoggingHandler dataflowLoggingHandler =
           DataflowWorkerLoggingInitializer.getLoggingHandler();
       dataflowLoggingHandler.publish(this, logRecord);
-
-      if (shouldLogFullThreadDump(lullDuration)) {
```

Comment:
why is reportLull changing so much?  I don't think you call this since your using reportBundleLull instead.  It seems like this would just remain the same.

Reply:
We moved this content to StackTraceUtils to reuse the logic. We have also removed the full thread dump from here to avoid the spamming.

Reply:
I don't think we should change the full thread dump for message level.  It happens only if a single message is stuck for 20 minutes which seems like the pipeline is fully broken anyway.  I agree leave it off for bundle level.